vendor/assets/stylesheets/dvl/core/grid.scss

```diff
@@ -11,6 +11,9 @@
   padding: 0;
   list-style: none; /* 2 */
   font-size: 0; /* 3 */
+  @media screen and (min-width: $lapWidth) {
```

Comment:
I'm not sure I follow. Isn't the point of `rem` units that they're already relative to screen size?

I find this behavior to be confusing and adding unnecessary complexity. Is it necessary to have?